Sources/RxMoya/MoyaProvider+Rx.swift

```diff
@@ -33,8 +33,7 @@ public extension Reactive where Base: MoyaProviderType {
 
     /// Designated request-making method with progress.
     func requestWithProgress(_ token: Base.Target, callbackQueue: DispatchQueue? = nil) -> Observable<ProgressResponse> {
-        let progressBlock: (AnyObserver) -> (ProgressResponse) -> Void = { observer in
-            return { progress in
+        let progressBlock: (AnyObserver) -> (ProgressResponse) -> Void = { observer in { progress in
```

Comment:
How about using the short form here ?

`let progressBlock = AnyObserver<ProgressResponse>.onNext`

Reply:
this is nice, thanks @hamdshah!